src/Components/Publishing/Header/NewsHeadline.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import styled from "styled-components"
+import { Fonts } from "../Fonts"
+
+interface NewsHeadlineProps {
+  title: string
+  article?: any
+}
+
+export const NewsHeadline: React.SFC<NewsHeadlineProps> = props => {
+  const { title } = props
+  return (
+    <NewsHeadlineParent>
+      <NewsHeadlineContainer>
+        <Title>{title}</Title>
+      </NewsHeadlineContainer>
+    </NewsHeadlineParent>
+  )
+}
+
+const NewsHeadlineParent = styled.div`
+  margin: 0 40px;
+`
+
+const NewsHeadlineContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 780px;
+  width: 100%;
+  margin: 40px auto;
+  box-sizing: border-box;
```

Comment:
Tbh not sure what `box-sizing` does; copied from `StandardHeader`.

Reply:
Ooh yeah box-sizing is interesting -- https://css-tricks.com/box-sizing/ and https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

You _might_ not need it here but maybe remove it and see if things change. It will be pretty obvious if things break because this controls sizing of content based on width/height/margin/padding